src/GitVersionCore/Log/NullLog.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+
+namespace GitVersion.Log
```

Comment:
Using [gerund](https://en.wikipedia.org/wiki/Gerund) form on words in namespaces avoid collision with class names. Since there could very well exist a class called `Log`, I therefore think `Logging` instead of `Log` is preferable as the namespace.

Reply:
I changed the namespace to Logging